pkg/client/v1alpha2/clientset/versioned/typed/ako/v1alpha2/fake/fake_l4rule.go

```diff
@@ -36,9 +35,9 @@ type FakeL4Rules struct {
 	ns   string
 }
 
-var l4rulesResource = schema.GroupVersionResource{Group: "ako.vmware.com", Version: "v1alpha2", Resource: "l4rules"}
+var l4rulesResource = v1alpha2.SchemeGroupVersion.WithResource("l4rules")
```

Comment:
Not sure why this is changed, if changes are related to l7Rules

Reply:
Autogeneration is done for  all crds in v1aplha2 version. So this part in L4Rule and SS0Rule is also changed.